### PR TITLE
Add A & AAAA record support

### PR DESCRIPTION
**Pull request by jlucidar**
If you register a new service, the addresses' field of your service is now being populated with the IP addresses of the machine.

### CODE REVIEW

**Comment by watson** [comment]
@jlucidar Thanks a lot for the pull request 😃

There's a few tests that are failing related to styling. Please update the styling to match the styling used so that the tests parse.

**Comment by jlucidar** [comment]
Hi @watson,
Everything looks good now. I modified the test to include the new changes.

File: test/bonjour.js [hunk 1]
```diff
@@ -135,6 +151,7 @@ test('bonjour.findOne - emitter', function (bonjour, t) {
     var browser = bonjour.findOne({ type: 'test' })
     browser.on('up', function (s) {
       t.equal(s.name, 'Emitter')
+      browser.stop()
```
Comment (watson) [inline test/bonjour.js]: How come you added the `browser.stop()` calls a few places? Did the event loop hang?

Comment (jlucidar) [inline test/bonjour.js]: no, it was one of my tentative to fix a bug that I didn't clear ...  I can remove it if you want me to.  
for whatever reason, the [buildServicesFor](https://github.com/watson/bonjour/blob/master/lib/browser.js#L134) function acted weirdly during the test; it was pushing duplicate addresses to the addresses field.
I resolved it, but I can't really explain why it was doing that.

Comment (watson) [inline test/bonjour.js]: no problem - I removed it my self :)

**Comment by watson** [comment]
@jlucidar thanks again for the PR 😄 It's now published as v3.4.0 on npm

**Pull request merged**